modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/AssetManagerImpl.java

```diff
@@ -1080,6 +1083,7 @@ private boolean isAuthorized(final String mediaPackageId, final String action) {
                 .collect(Collectors.toList());
         return getDatabase().selectProperties(mediaPackageId, SECURITY_NAMESPACE).parallelStream()
                 .map(p -> p.getId().getName())
+                .filter(p -> action.equals(getActionFromPropertyName(p)))
```

Comment:
Simpler and more efficient still would probably be to just use [`String:endsWith`](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#endsWith-java.lang.String-):
```suggestion
                .filter(p -> p.endsWith(action))
```

Reply:
You're totally right, and this simplifies the code a lot.